scripts/run-pr.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ -z "$1" ]; then
+  echo "No pull request ID was specified."
+  exit 1
+fi
+
+git fetch origin pull/${1}/head
```

Comment:
I can confirm that, nice catch. Mine is configured as `upstream`, as `origin` is my github fork (and default push target).